joebvp/VPmeasure.py

```diff
@@ -136,14 +99,16 @@ def setData(self,index,value,role=Qt.EditRole):
                 elif column == 6: self.fiterrors[2][row] = val
                 elif column == 7: self.fitpars[4][row] = val
                 elif column == 8: self.fiterrors[4][row] = val
+                elif column == 9: pass
```

Comment:
Why pass? (maybe is failing because the type of the variable?) could it be non str type?